### PR TITLE
@orta => [Metaphysics Logger] Fix some edge cases

### PR DESCRIPTION
**Pull request by mzikherman**
Noticed these, was actually causing Storybooks to crash locally.

### CODE REVIEW

File: src/Artsy/Relay/middleware/metaphysicsExtensionsLoggerMiddleware.ts [hunk 1]
```diff
@@ -23,7 +23,7 @@ export function metaphysicsExtensionsLoggerMiddleware() {
         const apis = _.omit(requests, ["stitching"])
         const requestCount = Object.keys(apis)
           .map(k => Object.keys(requests[k].requests).length) // API requests
-          .reduce((a, c) => a + c) // Add them all up
+          .reduce((a, c) => a + c, 0) // Add them all up
```
Comment (mzikherman) [inline src/Artsy/Relay/middleware/metaphysicsExtensionsLoggerMiddleware.ts]: Default value.

**Comment by orta** [comment]
Perfect - thanks

**Comment by orta** [comment]
Looks good /cc @l2succes

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 7d65ac49e70a0e1c44a4db7cd08e876250c1bde8

https://deploy-preview-2008--artsy-reaction.netlify.com

**Comment by mzikherman** [comment]
Now looks like:
screen shot 2019-02-08 at 5 53 34 pm

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit a0bc61364bfd92af99e63debb2c1e3099286d382

https://deploy-preview-2008--artsy-reaction.netlify.com

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit f19f8e191da945dcba9cc5f7937f71eee9b7ed87

https://deploy-preview-2008--artsy-reaction.netlify.com

**Comment by orta** [comment]
Perfect! Thankssssss

**Comment by artsyit** [comment]
:rocket: PR was released in v12.1.30 :rocket:

**Pull request merged**